core-api/src/main/java/com/sequenceiq/cloudbreak/validation/Permission.java

```diff
@@ -58,11 +61,35 @@
 
     private String description;
 
-    Permissions(Resource resource, Action action, String description) {
+    Permission(Resource resource, Action action, String description) {
         this.resource = resource;
         this.action = action;
         this.description = description;
-        this.name = resource.name() + ":" + action.name();
+        this.name = getName(resource, action);
+    }
+
+    public static String getName(Resource resource, Action action) {
+        return resource.name() + ':' + action.name();
+    }
+
+    public static boolean hasPermission(Set<String> permissions, Resource resource, Action action) {
+        if (resource == ORG) {
+            if (action == INVITE && permissions.contains(ORG_INVITE.name)) {
```

Comment:
BUG: if you dont have INVITE permission but you have MANAGE, then you should be able to invite others